src/environments/environment.prod.ts

```diff
@@ -1,6 +1,7 @@
 export const environment = {
   production: true,
   apiHost: "https://submission-dev.ebi.ac.uk/api/",
+  apiTokenRefresh: "https://explore.api.aai.ebi.ac.uk/token",
```

Comment:
Is there a better way to handle this constant variable?
The `authenticationHost ` has the same root url. If that would change then we would have to change both of them. The `apiTokenRefresh ` could be just a path under the root URI.